packages/lab/src/Flow/Node/Node.tsx

```diff
@@ -53,12 +45,16 @@ export interface HvFlowNodeProps<T = any> extends HvFlowBaseNodeProps<T> {
   actionsIconOnly?: HvActionsGenericProps["iconOnly"];
   /** Node maximum number of actions visible. */
   maxVisibleActions?: HvActionsGenericProps["maxVisibleActions"];
+  /** Node subtitle - this is typically the node "name" */
+  subtitle?: string;
+  /** Node group ID */
+  groupId?: string;
+  /** Node ID to match with the `nodeGroups`'s item */
```

Comment:
I don't know if I'm missing something, but the `nodeId` doesn't seem to be used anywhere. We should remove it.

Reply:
it's not 👍